pkg/obfuscator/mac_address.go

```diff
@@ -58,7 +58,7 @@ func NewMacAddressObfuscator(replacementType schema.ObfuscateReplacementType) (O
 
 	reporter := NewSimpleTracker()
 	// creating a new generator object
-	generator := newGenerator(consistentMACTemplate, staticMacReplacement)
+	generator := newGenerator(consistentMACTemplate, staticMacReplacement, 9999999999)
```

Comment:
"maximumSupportedObfuscations" can be used in order to avoid hard-coding

Reply:
split it out into another constant to not accidentally mix things up